src/main/java/uk/gov/register/thymeleaf/ThymeleafView.java

```diff
@@ -11,10 +12,12 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.nio.charset.StandardCharsets;
+import java.util.Optional;
 
 public class ThymeleafView extends View {
     protected final RequestContext requestContext;
     private String thymeleafTemplateName;
+    private final MarkdownProcessor markdownProcessor = new MarkdownProcessor();
```

Comment:
Now markdownProcessor object is created here so I think we can use this object in HomePageView class also.